packages/steam/index.ts

```
@@ -0,0 +1,347 @@
+export interface SteamAuthClientOptions {
+    socketPath?: string;
+}
+
+/**
+ * SteamAuthClient provides methods to authenticate with Steam
+ * through a C# service over Unix sockets.
+ */
+export class SteamAuthClient {
+    private socketPath: string;
+    private activeSocket: Socket | null = null;
+    private eventListeners: Map<string, Function[]> = new Map();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)